test/e2e/template_test.go

```diff
@@ -99,7 +99,7 @@ data:
 ` + depYAML
 
 	expectedYAML1Diff := `
-@@ create configmap/config-ver-1 (v1) namespace: kapp-test @@
+@@ create configmap/config-ver-1 (v1) namespace: ` + env.Namespace + ` @@
```

Comment:
How about using `strings.Replace()` to replace the occurrences of test namespace?

Reply:
Nice catch. Yes, `strings.Replace()` would be a better approach

Reply:
Same as above
`
Also, I think it would be good to use something like <e2e-test-ns> instead of kapp-test and then replace it with env.Namespace so that it is evident while reading the yaml that this value will be replaced.
`